### ISSUE TITLE
Fail to build libaco

### ISSUE THREAD

**Issue by xiaozongyang** [hnes/libaco#18]
When I build the project libaco by  executing command `bash make.sh -o no-m32 -o no-valgrind` I got the error
``` 
test_aco_benchmark.c:106:17: error: implicit declaration of function 'clock_gettime' is invalid in C99
      [-Werror,-Wimplicit-function-declaration]
    assert(0 == clock_gettime(CLOCK_MONOTONIC, &tstart));
test_aco_benchmark.c:168:31: error: use of undeclared identifier 'CLOCK_MONOTONIC'
    assert(0 == clock_gettime(CLOCK_MONOTONIC, &tstart));
```
And here is my envonment:
- OS：Mac OS Elcaption
- make: GNU Make 4.2.1
- compiler: Apple LLVM version 8.0.0 (clang-800.0.42.1)

Did I missing any key point?

**Comment by hnes** [comment]
Thank you very much for your report, @XiaoZYang ;-)

Actually, it is a known issue:

> In effect, `clock_gettime ` seems not to be implemented for macOS before Sierra 10.12.
>
> -- from this [link](https://stackoverflow.com/questions/5167269/clock-gettime-alternative-in-mac-os-x)

I would add such compliant support for "macOS before Sierra 10.12" as soon as I get enough spare time. Maybe you could choose to write your own patch (or just choose another platform like linux to build the benchmark) before that and PR is always welcome :D

PS:

The `clock_gettime` now is only required by the benchmark file `test_aco_benchmark.c`.

**Comment by xiaozongyang** [comment]
@hnes thanks for your explaining. I will try to build on Linux later.
